api/HTMLLegendElement.json

```diff
@@ -66,7 +66,7 @@
               "version_added": "4"
             },
             "ie": {
-              "version_added": "≤6"
+              "version_added": "6"
```

Comment:
Let's assume this is correct, I did in https://github.com/mdn/browser-compat-data/pull/8972.